package.json

```diff
@@ -54,7 +54,7 @@
     "README.md"
   ],
   "devDependencies": {
-    "@types/chai": "^4.2.3",
+    "@types/chai": "^4.2.0",
```

Comment:
Why are we downgrading `@types/chai` ?

Reply:
We specify typescript 3.6.3. There seems to be an issue with typescript 3.6.3 icw the chai 4.2.3 ts definitions:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/38473

 We should upgrade typescript, but let's not do that in this MR.